src/goto-programs/goto_trace.cpp

```diff
@@ -129,22 +129,25 @@ void goto_trace_stept::output(
 /// \param expr: expression to get numeric representation from
 /// \param options: configuration options
 /// \return a string with the numeric representation
-static std::string
-numeric_representation(const exprt &expr, const trace_optionst &options)
+static std::string numeric_representation(
+  const constant_exprt &expr,
+  const trace_optionst &options)
 {
   std::string result;
   std::string prefix;
+
+  const irep_idt &value = expr.get_value();
```

Comment:
```suggestion
  const std::string &value = id2string(expr.get_value());
```
and remove the repeated `id2string` below.

Reply:
Uh, no, we are trying to change the signature of bv2integer to irep_idt.